deploy-config/demo.yml

```
@@ -1,8 +1,10 @@
 env: demo
 instances: 1
 memory: 1G
+command: gunicorn -c /home/vcap/app/gunicorn_config.py application
```

Ah, this command should retain the New Relic config from the original manifest!  Apologies for the confusion, it should look like this:

```suggestion
command: newrelic-admin run-program gunicorn -c /home/vcap/app/gunicorn_config.py application
```

That's on me for not fully explaining before and not having the other sandbox changes in in time for this. 😆